ThirdPartyAdapters/liftoffmonetize/liftoffmonetize/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -346,62 +348,46 @@ public void onAdLeftApplication(@NonNull BaseAd baseAd) {
     }
   }
 
-  private void createBanner() {
-    if (bannerAd == null) {
-      AdError error = new AdError(ERROR_VUNGLE_BANNER_NULL,
-          "Try to play banner ad but the Vungle BannerAd instance not created.",
-          ERROR_DOMAIN);
-      Log.d(TAG, error.toString());
-      if (mediationBannerListener != null) {
-        mediationBannerListener.onAdFailedToLoad(VungleInterstitialAdapter.this, error);
-      }
-      return;
-    }
-
-    View bannerView = bannerAd.getBannerView();
-    // The Vungle SDK performs an internal check to determine if a banner ad is playable.
-    // If the ad is not playable, such as if it has expired, the SDK will return `null` for the
-    // banner view.
-    if (bannerView == null) {
-      AdError error = new AdError(ERROR_VUNGLE_BANNER_NULL,
-          "Vungle SDK returned a successful load callback, but getBannerView() returned null.",
-          ERROR_DOMAIN);
-      Log.d(TAG, error.toString());
-      if (mediationBannerListener != null) {
-        mediationBannerListener.onAdFailedToLoad(VungleInterstitialAdapter.this, error);
-      }
-      return;
-    }
-
-    // Add rules to ensure the banner ad is located at the center of the layout.
-    RelativeLayout.LayoutParams adParams = new RelativeLayout.LayoutParams(
-        RelativeLayout.LayoutParams.WRAP_CONTENT, RelativeLayout.LayoutParams.WRAP_CONTENT);
-    adParams.addRule(RelativeLayout.CENTER_HORIZONTAL, RelativeLayout.TRUE);
-    adParams.addRule(RelativeLayout.CENTER_VERTICAL, RelativeLayout.TRUE);
-    bannerView.setLayoutParams(adParams);
-    bannerLayout.addView(bannerView);
-    if (mediationBannerListener != null) {
-      mediationBannerListener.onAdLoaded(VungleInterstitialAdapter.this);
-    }
-  }
-
   @NonNull
   @Override
   public View getBannerView() {
     Log.d(TAG, "getBannerView # instance: " + hashCode());
     return bannerLayout;
   }
 
-  public static BannerAdSize getVungleBannerAdSizeFromGoogleAdSize(Context context, AdSize adSize) {
+  public static VungleAdSize getVungleBannerAdSizeFromGoogleAdSize(AdSize adSize,
+      String placementId) {
+    VungleAdSize vngAdSize;
+    if (VungleAds.isInline(placementId)) {
```

Comment:
For iOS side, this logic is within the SDK side?  We don't need to align?
cc @ManojBudumuru-vungle

Reply:
Thanks, missed to remove this line. Updated.